includes/gateways/paypal/scripts.php

```diff
@@ -32,24 +32,14 @@ function maybe_enqueue_polyfills() {
 
 	global $wp_version;
 	if ( version_compare( $wp_version, '5.0', '>=' ) ) {
-		wp_enqueue_script( 'wp-polyfill-promise' );
-		wp_enqueue_script( 'wp-polyfill-fetch', '', array( 'wp-polyfill-promise' ) );
-		wp_enqueue_script( 'wp-polyfill-formdata' );
+		wp_enqueue_script( 'wp-polyfill' );
 	} else {
 		wp_enqueue_script(
-			'promise-polyfill',
-			EDD_PLUGIN_URL . 'assets/js/promise-polyfill.min.js',
+			'wp-polyfill',
```

Comment:
I set line 34 to `false` to trigger the else and PayPal is working both ways in IE now.

I'm having intermittent issues with IE when the PayPal window closes--the checkout doesn't always complete on the first try. Clicking the PayPal button again takes me through the checkout process and it seems to complete fine. This does, however, result in two EDD transactions, one complete and one pending. Only one is recorded in PayPal. The logs for this pair of orders show:

```
2021-5-20 19:49:34 - PayPal - create_order()
2021-5-20 19:49:36 - -- Successful PayPal response. PayPal order ID: 10E36428WE287632Y; EDD order ID: 416
2021-5-20 19:51:58 - PayPal - create_order()
2021-5-20 19:51:59 - -- Successful PayPal response. PayPal order ID: 5KB63728FM042244R; EDD order ID: 417
2021-5-20 19:52:23 - PayPal - capture_order()
2021-5-20 19:52:25 - -- PayPal Response code: 201; order ID: 5KB63728FM042244R
```

Nothing is logged in the error log. I don't know if there is a way to tell if this is IE bugging, or PayPal, or us, or just the alignment of the stars. I'll try again tomorrow as well.

Reply:
Anything in the debug log? Can also check payment gateway errors (Downloads > Reports > Logs > Payment Errors dropdown)

Reply:
Nope, nothing in the browser console and no payment gateway errors. I'll try some more purchases today.

Reply:
No errors, but I appear to be able to reliably trigger it by not being logged into PayPal. If I'm already logged in, then the purchase goes through smoothly. If I'm not logged in, and log in through the popup window, or if I'm logged in and click the "Not you?" link to log out and then force the log in again, when I come back to the checkout screen, nothing happens. So even if I don't click the "remember me" checkbox, when I do a second attempt, PayPal considers it to be part of the same session, I guess, and I'm already logged in, and everything works. But if I have to log in, then the process hangs in between create_order and capture_order, apparently.